itwm_example/tests/probe_classes/data_source.py

```diff
@@ -0,0 +1,25 @@
+import unittest
+
+from force_bdss.api import DataValue
```

Comment:
Being pedantic, Enthought style guidelines ask for a new line between module imports from external packages and this package. I've added them to the new starter document for future reference 

```suggestion
from force_bdss.api import DataValue

```

Reply:
That's my fault, thank you for pointing that out.